docs/CNDP_README.md

```diff
@@ -0,0 +1,421 @@
+<!--
+SPDX-License-Identifier: Apache-2.0
+Copyright 2020 Intel Corporation
+-->
+
+# **Cloud Native Data Plane (CNDP)**
+
+Cloud Native Data Plane (CNDP) is a collection of user space libraries for accelerating packet processing for cloud applications. It aims to provide better performance than that of standard network socket interfaces by using an I/O layer primarily built on AF_XDP, an interface that delivers packets directly to user space, bypassing the kernel networking stack. For more details refer https://cndp.io/
+
+CNDP BESS port enables sending/receiving packets to/from network interface using AF-XDP.  CNDP integration with OMEC BESS UPF enables a software-based datapath and also provides deployment flexibility for kubernetes based deployments where CNDP uses [AF-XDP device plugin](https://github.com/intel/afxdp-plugins-for-kubernetes).
+
+### Table Of Contents
+  * [Step 1: Build the OMEC UPF docker container](#build-docker-image)
+  * [Step 2: Test setup](#test-setup)
+  * [Step 3: Run UPF in System 1](#run-upf)
+  * [Step 4: Run DPDK packet generator in System 2](#run-dpdk-packet-generator)
+  * [CNDP OMEC-UPF multiple worker threads setup](#upf-multiple-workers-setup)
```

Comment:
The links for the individual sections are not correct. For example, the link for `step 1` should be `#step-1-build-the-omec-upf-docker-container`, the link for `step 2` should be `#step-2-test-setup`, and so on...

Reply:
Ok. How can I verify this.  "Step1: " has a colon. So can the colon be replaced by hyphen in section name ?